pylint/message/message_id_store.py

```diff
@@ -120,4 +120,9 @@ def get_active_msgids(self, msgid_or_symbol: str) -> List[str]:
                 msgid_or_symbol=msgid_or_symbol
             )
             raise UnknownMessageError(error_msg)
-        return [msgid]
+        # logging.debug(
+        #    "Return for {} and msgid {} is {}".format(
+        #        msgid_or_symbol, msgid, self.__old_names.get(msgid, [msgid])
+        #    )
+        # )
+        return self.__old_names.get(msgid, [msgid])
```

Comment:
This line is the one that makes multiple messages with the same old name possible, everything else is tests or optimizations.